app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -116,9 +118,12 @@ private void setUpToolbarToggle() {
 
     private void replaceFrameWithFragment(Fragment fragment){
         FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
-        transaction.replace(R.id.frame, fragment);
+        transaction.replace(R.id.frame_layout, fragment);
         transaction.commit();
     }
+    public void openDrawer(){
+        drawerLayout.openDrawer(GravityCompat.START);
```

Comment:
Nit-picking: Use a static import for this.  This should read `drawerLayout.openDrawer(START);`. See above for the import statement.